src/IdentityManager/WebApi/Controllers/AccountController.cs

```diff
@@ -8,16 +8,21 @@
 using IdentityManager.WebApi.Models;
 using Travely.IdentityManager.API.Identity;
 using IdentityManager.WebApi.Models.Request;
+using Microsoft.AspNetCore.Http;
+using IdentityManager.WebApi.Models.Error;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.JsonPatch;
+using IdentityManager.WebApi.Extensions;
 
 namespace Travely.IdentityManager.WebApi.Controllers
 {
     [Route("api/[controller]")]
-    [ApiController]
+    [Produces("application/json")]
+    //[ApiController]
```

Comment:
Uncomment `ApiController`. 
https://www.strathweb.com/2018/02/exploring-the-apicontrollerattribute-and-its-features-for-asp-net-core-mvc-2-1/